org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -1567,6 +1571,23 @@ Sso8dxfh53Admin:
     principalId: !Ref 8dxfh53AdminGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-admin-permission-set-arn' ]
 
+SsoBuA2aDwAdmin:
+  Type: update-stacks
+  DependsOn: SsoAdministrator
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.11/templates/SSO/aws-sso.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-8dxfh53-admin'
```

Comment:
The stack name needs to be updated with the new random string

Reply:
fixed